apps/style/common.scss

```diff
@@ -268,20 +269,25 @@ button.arrow:disabled {
 }
 
 .soft-buttons-1 {
-  display: table-cell;
-  vertical-align: top;
+  @supports not (gap: 9px){
```

Comment:
nit: there should be a space between `) {` (ideally, we should have a linter do this for us -- i'm surprised we don't)